app/src/main/kotlin/com/simplemobiletools/keyboard/views/MyKeyboardView.kt

```diff
@@ -1396,9 +1396,7 @@ class MyKeyboardView @JvmOverloads constructor(context: Context, attrs: Attribut
                 // If we're not on a repeating key (which sends on a DOWN event)
                 if (mRepeatKeyIndex == NOT_A_KEY && !mMiniKeyboardOnScreen && !mAbortKey) {
                     detectAndSendKey(mCurrentKey, touchX, touchY, eventTime)
-                }
-
-                if (currentKeyCode == KEYCODE_SPACE && !mIsLongPressingSpace) {
+                } else if (currentKeyCode == KEYCODE_SPACE && !mIsLongPressingSpace) {
```

Comment:
Also, I have found another bug with these two if statements. In number input layout, on every space click it prints two space symbols. So I had to add else if here, in order to fix that. We need to test it now, and all should be awesome